Source/UI/PaymentForm/View/BillingFormSummaryView.swift

```diff
@@ -59,7 +59,6 @@ class BillingFormSummaryView: UIView {
 
         titleLabel.update(with: style.title)
         hintLabel.update(with: style.hint)
-        summaryLabel.update(with: style.hint)
```

Comment:
This method already gets called with correct style passed  on line 65 so this was doing an extra refresh with wrong style